src/test/java/redis/clients/jedis/examples/GeoShapeFieldsUsageInRediSearch.java

```diff
@@ -0,0 +1,106 @@
+package redis.clients.jedis.examples;
+
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Polygon;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKTReader;
+
+import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.JedisPooled;
+import redis.clients.jedis.UnifiedJedis;
+import redis.clients.jedis.search.FTSearchParams;
+import redis.clients.jedis.search.SearchResult;
+import redis.clients.jedis.search.schemafields.GeoShapeField;
+
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static redis.clients.jedis.search.RediSearchUtil.toStringMap;
+
+/**
```

Comment:
/**
 * As of RediSearch 2.8.4, advanced GEO querying with GEOSHAPE fields is supported.
 *
 *Any object/library producing
 * <a href="https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry">
 * in a well-known text (WKT)</a> in {@code toString()} method can be used.
 *
 * This example uses the
 * <a href="https://github.com/locationtech/jts">JTS</a> library.
 * <pre>
 * {@code
 * <dependency>
 *   <groupId>org.locationtech.jts</groupId>
 *   <artifactId>jts-core</artifactId>
 *   <version>1.19.0</version>
 * </dependency>
 * }
 * </pre>
 */